File: Ass_14/Assignment_5.c
#include<stdio.h>

void Pattern(int iRow, int iCol)
{
    int i = 0;
    int j= 0;
    int No = 0;

    for(i = 1 ;i <= iRow; i++)
    {
        if(i % 2 != 0)
        {
            No = i;
            for(j = 1 ; j <= iCol; j++)
            {
                printf("%d\t",No++);
            }
           
        }
       else
       {
            No = i;
            for(j = 1;j <= iCol; j++)
        {
            printf("%d\t",No++);
        }
        
       }
       printf("\n");

       
    }
}
int main()
{
    int iValue1 = 0;
    int iValue2 = 0;

    printf("Enter number of row:");
    scanf("%d",&iValue1);

    printf("Enter number of column:");
    scanf("%d",&iValue2);

    Pattern(iValue1,iValue2);

    return 0;
}

File: Ass_46/Assignment_5.java
import java.util.*;

class Pattern
{
    public void Pattern(int iRow,int iCol)
    {
        int i = 0, j = 0;
            
        int iNo = 1;
        for(i = 0;i <iRow;i++)
        {
            for(j = 0; j <iCol;j++)
            {
                System.out.print(iNo + " ");
                iNo++;

            }

            System.out.println("\n");

        }
        
    }
}
class Assignment_5
{
    public static void main(String[]args)
    {
        Scanner sobj = new Scanner(System.in);
        System.out.println("Enter the number of rows :");
        int iRow = sobj.nextInt();

        System.out.println("Enter the number of columns :");
        int iCol = sobj.nextInt(); 

        Pattern pobj = new Pattern();
        pobj.Pattern(iRow,iCol);
    }
}